README.md

```diff
@@ -40,7 +40,7 @@ The crypt only contains a few ciphers, but with time (and your help) it will hav
 - [ ] Porta
 - [ ] Homophonic
 - [ ] Four-Square
-- [ ] Playfair
+- [X] Playfair
```

Comment:
Once you merge in master, please move this completed cipher up so it's directly underneath the other completed cipher's in this list.